aiuser/settings/prompt.py

```diff
@@ -75,7 +75,10 @@ async def prompt_show(self, ctx: commands.Context, mention: Optional[Union[disco
             title = f"The prompt for {ctx.channel.mention if channel_prompt else 'this server'} is:"
 
         if not prompt:
-            prompt = DEFAULT_PROMPT
+            if self.config.custom_text_prompt:
+                prompt = self.config.custom_text_prompt
+            else:
+                prompt = DEFAULT_PROMPT
```

Comment:
```suggestion
            prompt = await self.config.custom_text_prompt() or DEFAULT_PROMPT
```
we can condense it a bit